src/styles/variables/colors.css

```diff
@@ -10,6 +10,11 @@
   --draw-steel-c-failure: red;
   --draw-steel-c-item-header-bg: #bebbab;
   --draw-steel-c-item-alternating-bg: rgba(0, 0, 0, .05);
+  --draw-steel-c-laserline: #856C4B;
+  --color-shadow-primary: #ccccccaa;
+  --button-focus-outline-color: var(--draw-steel-c-dark);
+  --button-hover-background-color: var(--draw-steel-c-dark);
+  /*--input-text-color:var(--draw-steel-c-black);*/
```

Comment:
Afraid this is not gonna work. Right now, this would bleed over to every part of the platform due to `:root` above.